modules/extensions-support/src/test/java/org/mule/module/extension/HeisenbergExtension.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension;
+
+import org.mule.api.MuleContext;
+import org.mule.api.MuleException;
+import org.mule.api.context.MuleContextAware;
+import org.mule.api.lifecycle.InitialisationException;
+import org.mule.api.lifecycle.Lifecycle;
+import org.mule.extension.ExtensionManager;
+import org.mule.extension.annotations.Extensible;
+import org.mule.extension.annotations.Extension;
+import org.mule.extension.annotations.Operations;
+import org.mule.extension.annotations.Parameter;
+import org.mule.extension.annotations.ParameterGroup;
+import org.mule.extension.annotations.capability.Xml;
+import org.mule.extension.annotations.param.Optional;
+
+import java.math.BigDecimal;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+@Extension(name = HeisenbergExtension.EXTENSION_NAME, description = HeisenbergExtension.EXTENSION_DESCRIPTION, version = HeisenbergExtension.EXTENSION_VERSION)
+@Operations({HeisenbergOperations.class, MoneyLaunderingOperation.class})
+@Xml(schemaLocation = HeisenbergExtension.SCHEMA_LOCATION, namespace = HeisenbergExtension.NAMESPACE, schemaVersion = HeisenbergExtension.SCHEMA_VERSION)
+@Extensible(alias = "heisenberg-empire")
+public class HeisenbergExtension implements Lifecycle, MuleContextAware
```

Comment:
This could be funny, but if somebody has to see the full serie in order to understand this example, then we are fried